src/olympia/amo/middleware.py

```diff
@@ -287,17 +287,24 @@ class SetRemoteAddrFromForwardedFor(MiddlewareMixin):
     Set REMOTE_ADDR from HTTP_X_FORWARDED_FOR if necessary.
 
     The request flow is either:
-    Client -> CDN -> Load balancer -> WSGI proxy -> addons-server.
+    Client -> CDN -> Load balancer -> WSGI proxy -> addons-server
     or
-    Client -> Load balancer -> WSGI proxy -> addons-server.
+    Client -> CDN -> CDN shield -> Load balancer -> WSGI proxy -> addons-server
+    or
+    Client -> Load balancer -> WSGI proxy -> addons-server
 
     Currently:
-    - CDN is CloudFront
+    - CDN is CloudFront or Fastly
+    - CDN shield is an additional PoP on the CDN that request can go through
```

Comment:
It's not clear exactly what "shielded" means in this context. Could you add a sentence explaining this?